### ISSUE TITLE
Do not show the language selection menu at every start

### ISSUE THREAD

**Issue by enrico3** [hsnborn22/LinGo#2]
Hello,
since most users will only learn one language at a time I think it would be better to save the currently studied language and add an option in the main menu to change the studied language. This way the user does not need to select his language at every start.

**Comment by hsnborn22** [comment]
Good idea; actually I was thinking about removing the language selecting menu altogether; what do you think about this?
